.github/workflows/stack-integration_tests.yml

```diff
@@ -85,6 +85,72 @@ jobs:
         run: |
           tox -e stack.test.integration
 
+  stack-integration-tests-public:
+    strategy:
+      max-parallel: 3
+      matrix:
+        os: [ubuntu-latest]
```

Comment:
Once this works can we enable it on all OSes?

Reply:
Porting the bash stuff to windows might take some time we can possibly add this as a task for later.